schemas/json/theme.json

```diff
@@ -1077,7 +1077,7 @@
 		"version": {
 			"description": "Version of theme.json to use.\nSince 5.8.",
```

Comment:
Hi, was looking and this and wondered if there a way to say: 

- this property `version` is present since 5.8
- its value was changed to `2` in 5.9

Reply:
I think trying to add notes to that level of detail would be too difficult to keep updated. We could also just remove those version notes completely since we have tagged versioning now. However, it's still kind of nice when using trunk to see when things were added without having to open all the previous versions.

Reply:
I _do_ think that we need to update the `"Version of theme.json to use.\nSince 5.8.",` to be `"Version of theme.json to use.\nSince 5.9.", correct? Because v2 of `theme.json` is not available until WordPress 5.9 release.

Reply:
Oh, I think I see my confusion. We're describing the property introductory version. Super confusing!

> We could also just remove those version notes completely since we have tagged versioning now.

I would vote for this based on my example of confusion. 🤦